files/zh-cn/web/api/inputevent/gettargetranges/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: InputEvent：getTargetRanges() 方法
+slug: Web/API/InputEvent/getTargetRanges
+l10n:
+  sourceCommit: 72ca3d725e3e56b613de3ac9727bd0d6d619c38a
+---
+
+{{APIRef("UI Events")}}
+
+{{domxref("InputEvent")}} 接口的 **`getTargetRanges()`** 方法返回一个 {{domxref("StaticRange")}} 对象数组，如果不取消输入事件，该数组受到 DOM 更改的影响。
```

Comment:
Good catch! Please create a PR to update to fix the macro-refer to "Static Range"

Reply:
https://github.com/mdn/content/pull/32910